src/main/amp/config/alfresco/subsystems/Transformers/default/remote-jod-transformers.properties

```diff
@@ -49,6 +49,11 @@ content.transformer.complex.RemoteJODConverter.Html2Pdf.pipeline=RemoteJODConver
 content.transformer.complex.RemoteJODConverter.Html2Pdf.extensions.html.pdf.supported=true
 content.transformer.complex.RemoteJODConverter.Html2Pdf.extensions.html.pdf.priority=50
 
+
+content.transformer.RemoteJODConverter.extensions.tiff.pdf.priority=30
```

Comment:
Do not set priority here. It is inherited from the default in `content.transformer.RemoteJODConverter.priority`